DependencyInjection/Configuration.php

```diff
@@ -63,6 +63,16 @@ public function getConfigTreeBuilder()
                             ->defaultValue(array('CREATE', 'LIST', 'DELETE', 'UNDELETE', 'EXPORT', 'OPERATOR', 'MASTER'))
                             ->prototype('scalar')->end()
                         ->end()
+                        ->scalarNode('role_admin')
+                            ->cannotBeEmpty()
+                            ->defaultValue('ROLE_SONATA_ADMIN')
+                            ->info('Role which will see the top nav bar and dropdown groups regardless of its configuration')
+                        ->end()
+                            ->scalarNode('role_super_admin')
+                            ->cannotBeEmpty()
+                            ->defaultValue('ROLE_SUPER_ADMIN')
+                        ->info('Role which will perform all admin actions, see dashboard, menu and search groups regardless of its configuration')
```

Comment:
Indention ist wrong

Reply:
I used the style of the original file, like https://github.com/gremo/SonataAdminBundle/blob/b12ab3fed7f81870e67bce3945393b4f9a1b03dc/DependencyInjection/Configuration.php#L128... where is wrong btw?

Reply:
There is one tab missing in front of `->info...`